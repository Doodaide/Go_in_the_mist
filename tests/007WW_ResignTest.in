who
place A10
who
place B10
who
resign