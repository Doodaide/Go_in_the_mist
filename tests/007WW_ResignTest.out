B
W
B
White wins!
A10B10
Thank you for playing!
